src/pip/_internal/utils/compat.py

```diff
@@ -210,13 +224,15 @@ def samefile(file1, file2):
 
 if hasattr(shutil, 'get_terminal_size'):
     def get_terminal_size():
+        # type: (...) -> Tuple
         """
         Returns a tuple (x, y) representing the width(x) and the height(y)
         in characters of the terminal window.
         """
-        return tuple(shutil.get_terminal_size())
+        return tuple(shutil.get_terminal_size())  # type: ignore
 else:
     def get_terminal_size():
+        # type: (...) -> Tuple
```

Comment:
```suggestion
        # type: () -> Tuple
```

The `...` isn't needed here.